combat-trainer.lic

```diff
@@ -152,6 +152,7 @@ class LootProcess
   include DRC
   include DRCI
   include DRCS
+  include DRCH
```

Comment:
Is this necessary as part of this PR, or is this a line leftover from the necro healing changes?